test/test_ranges.cpp

```diff
@@ -233,6 +233,10 @@ TEST_CASE("ranges fill") {
     std::string b;
     fill('b') | first_n(5) | append(b);
     CHECK(b == "bbbbb");
+
+    int v = 7;
+    CHECK((fill(v) | take(5) | sum()) == 7*5);
```

Comment:
`... == v*5` or else the use of `v` is kinda useless

Reply:
v is entirely useless in there, but it should still work.

Reply:
Why not use it in the test for clarity, though? :)